.github/workflows/rosa-run-benchmark.yml

```diff
@@ -5,14 +5,38 @@ on:
     inputs:
       clusterName:
         description: 'Name of the cluster'
-        type: text
+        type: string
+      namespacePrefix:
+        description: 'Openshift Project name prefix'
+        type: string
       replicas:
         description: 'Number of keycloak pods'
         required: true
         default: '3'
+      disableStickySessions:
+        description: 'Disable sticky session in Openshift Route'
+        type: boolean
+        default: false
+        required: true
+      image:
+        description: 'Keycloak Image'
+        type: string
+      podMemoryLimit:
+        description: 'Keycloak pod memory limit (in MB)'
```

Comment:
we would need to set some default values, if we are going to introduce limits, also consider adding requests, as requests and limits go hand in hand.